gulpfile.babel.js

```diff
@@ -0,0 +1,19 @@
+import gulp     from 'gulp';
```

Comment:
gulp files tend to be less verbose and more clear than gruntfiles. (though i've certainly seen bad examples to argue the other way).
I try keep the overall of what we're doing in one place (`gulpfile*.js`) - and each task defined in a separate files (tasks should be idempotent, and not depend on each other or external state).